Lib/json/__init__.py

```diff
@@ -272,6 +272,16 @@ def detect_encoding(b):
     return 'utf-8'
 
 
+def from_file(filepath, *, cls=None, object_hook=None, parse_float=None,
+        parse_int=None, parse_constant=None, object_pairs_hook=None, **kw):
+    with open(filepath) as _data:
+        data = _data
```

Comment:
What's going on here?! You copy the file object out, then the `with` closes it (hopefully), then you try to read from the closed file?